### ISSUE TITLE
Sort not work with not semantic versions

### ISSUE THREAD

**Issue by RCristiano** [victorgarric/pip_search#20]
```sh
pip_search -s version bs4

Traceback (most recent call last):
  File "/home/rcristiano/.local/bin/pip_search", line 8, in <module>
    sys.exit(main())
  File "/home/rcristiano/.local/pipx/venvs/pip-search/lib64/python3.10/site-packages/pip_search/__main__.py", line 15, in main
    search(query=' '.join(args.query), opts=args)
  File "/home/rcristiano/.local/pipx/venvs/pip-search/lib64/python3.10/site-packages/pip_search/pip_search.py", line 30, in search
    snippets = sorted(
  File "/home/rcristiano/.local/pipx/venvs/pip-search/lib64/python3.10/site-packages/pip_search/pip_search.py", line 32, in <lambda>
    key=lambda s: StrictVersion(s.select_one(
  File "/usr/lib64/python3.10/distutils/version.py", line 40, in __init__
    self.parse(vstring)
  File "/usr/lib64/python3.10/distutils/version.py", line 137, in parse
    raise ValueError("invalid version number '%s'" % vstring)
ValueError: invalid version number '5.1.3.0'
```

**Comment by RCristiano** [comment]
https://docs.python.org/3/whatsnew/3.10.html#distutils-deprecated

https://www.python.org/dev/peps/pep-0386/

**Comment by RCristiano** [comment]
Solved in #19
